couch2pg/src/db.js

```diff
@@ -0,0 +1,44 @@
+const { COUCHDB_SECURE, COUCHDB_USER, COUCHDB_PASSWORD, COUCHDB_HOST, COUCHDB_PORT, COUCHDB_DBS } = process.env;
+const
+  {
+    POSTGRES_SCHEMA,
+    POSTGRES_TABLE,
+    POSTGRES_USER,
+    POSTGRES_PASSWORD,
+    POSTGRES_HOST,
+    POSTGRES_DB,
+    POSTGRES_PORT
+  } = process.env;
+
+const PG_OPTS = {
+  user: POSTGRES_USER,
+  password: POSTGRES_PASSWORD,
+  host: POSTGRES_HOST,
+  port: POSTGRES_PORT,
+  database: POSTGRES_DB,
+};
+
+import PouchDb from 'pouchdb-core';
+import http from 'pouchdb-adapter-http';
+import session from 'pouchdb-session-authentication';
+
+PouchDb.plugin(http);
+PouchDb.plugin(session);
+
+import pg from 'pg';
+
+export const postgresTable = `${POSTGRES_SCHEMA}.${POSTGRES_TABLE}`;
+export const postgresProgressTable = `${POSTGRES_SCHEMA}.couchdb_progress`;
+export const postgresSchema = POSTGRES_SCHEMA;
+
+export const getPgClient = async () => {
```

Comment:
An optimisation to consider is using connection [pooling](https://node-postgres.com/features/pooling). Not a blocker for this PR for now though.

Reply:
I tried this before and the pool failed randomly. i think we know too little about pooling to use it without surprises. Can we reconsider this at a later time?

Reply:
i really appreciate it.